diff -up nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkImgTransformationsPlugin.cpp.omv~ nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkImgTransformationsPlugin.cpp
--- nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkImgTransformationsPlugin.cpp.omv~	2024-06-01 21:21:58.808536581 +0200
+++ nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkImgTransformationsPlugin.cpp	2024-06-01 21:22:06.300610826 +0200
@@ -34,6 +34,7 @@
 #include "DkToolbars.h"
 
 #include <QMouseEvent>
+#include <QActionGroup>
 
 #define PI 3.14159265
 
diff -up nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkImgTransformationsPlugin.h.omv~ nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkImgTransformationsPlugin.h
--- nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkImgTransformationsPlugin.h.omv~	2024-06-01 21:20:27.207631291 +0200
+++ nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkImgTransformationsPlugin.h	2024-06-01 21:20:45.939816031 +0200
@@ -268,8 +268,8 @@ class DkInteractionRects : public QWidge
 
 public:
 	
-	DkInteractionRects(QRect imgRect, QWidget* parent = 0, Qt::WindowFlags f = 0);
-	DkInteractionRects(QWidget* parent = 0, Qt::WindowFlags f = 0);
+	DkInteractionRects(QRect imgRect, QWidget* parent = 0, Qt::WindowFlags f = Qt::WindowFlags());
+	DkInteractionRects(QWidget* parent = 0, Qt::WindowFlags f = Qt::WindowFlags());
 	~DkInteractionRects();
 
 	void draw(QPainter* painter);
diff -up nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkSkewEstimator.cpp.omv~ nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkSkewEstimator.cpp
--- nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkSkewEstimator.cpp.omv~	2024-06-01 21:41:45.208610935 +0200
+++ nomacs-3.17.2295/ImageLounge/plugins/AffineTransformations/src/DkSkewEstimator.cpp	2024-06-01 21:41:49.164657431 +0200
@@ -301,7 +301,7 @@ cv::Mat DkSkewEstimator::computeEdgeMap(
 
 int DkSkewEstimator::randInt(int low, int high) {
 
-	return qrand() % ((high + 1) - low) + low;
+	return rand() % ((high + 1) - low) + low;
 }
 
 QVector<QVector3D> DkSkewEstimator::computeWeights(cv::Mat edgeMap, int direction) {
diff -up nomacs-3.17.2295/ImageLounge/plugins/CompositePlugin/src/SbChannelWidget.cpp.omv~ nomacs-3.17.2295/ImageLounge/plugins/CompositePlugin/src/SbChannelWidget.cpp
--- nomacs-3.17.2295/ImageLounge/plugins/CompositePlugin/src/SbChannelWidget.cpp.omv~	2024-06-01 21:18:50.394680123 +0200
+++ nomacs-3.17.2295/ImageLounge/plugins/CompositePlugin/src/SbChannelWidget.cpp	2024-06-01 21:19:11.566887586 +0200
@@ -37,8 +37,8 @@ namespace nmc {
 		if (couldLoad) {
 			QImage qImg = bl.image();
 			if (qImg.hasAlphaChannel()) {
-				emit newAlpha(qImg.alphaChannel());		//aparently alphaChannel() is obsolete, however I don't know what else to use
-				//emit newAlpha(qImg.convertToFormat(QImage::Format_Alpha8));		//because that doesn't seem to work as expected..
+				//emit newAlpha(qImg.alphaChannel());		//aparently alphaChannel() is obsolete, however I don't know what else to use
+				emit newAlpha(qImg.convertToFormat(QImage::Format_Alpha8));		//because that doesn't seem to work as expected..
 			}
 			else {
 				emit newAlpha(QImage());
@@ -170,4 +170,4 @@ namespace nmc {
 			emit imageChanged(c);
 		}
 	}
-};
\ No newline at end of file
+};
diff -up nomacs-3.17.2295/ImageLounge/plugins/FakeMiniaturesPlugin/src/DkFakeMiniaturesDialog.cpp.omv~ nomacs-3.17.2295/ImageLounge/plugins/FakeMiniaturesPlugin/src/DkFakeMiniaturesDialog.cpp
--- nomacs-3.17.2295/ImageLounge/plugins/FakeMiniaturesPlugin/src/DkFakeMiniaturesDialog.cpp.omv~	2024-06-01 21:44:26.226474371 +0200
+++ nomacs-3.17.2295/ImageLounge/plugins/FakeMiniaturesPlugin/src/DkFakeMiniaturesDialog.cpp	2024-06-01 21:44:33.018551799 +0200
@@ -477,7 +477,7 @@ void DkPreviewLabel::mouseReleaseEvent(Q
 	QLabel::mouseReleaseEvent(e);
 };
 
-void DkPreviewLabel::enterEvent(QEvent * e){
+void DkPreviewLabel::enterEvent(QEnterEvent * e){
 	showROI = true;
 	repaint();
     QLabel::enterEvent(e);
diff -up nomacs-3.17.2295/ImageLounge/plugins/FakeMiniaturesPlugin/src/DkFakeMiniaturesDialog.h.omv~ nomacs-3.17.2295/ImageLounge/plugins/FakeMiniaturesPlugin/src/DkFakeMiniaturesDialog.h
--- nomacs-3.17.2295/ImageLounge/plugins/FakeMiniaturesPlugin/src/DkFakeMiniaturesDialog.h.omv~	2024-06-01 21:44:34.562569388 +0200
+++ nomacs-3.17.2295/ImageLounge/plugins/FakeMiniaturesPlugin/src/DkFakeMiniaturesDialog.h	2024-06-01 21:44:43.322669099 +0200
@@ -196,7 +196,7 @@ protected:
     void mousePressEvent(QMouseEvent *e);
     void mouseMoveEvent(QMouseEvent *e);
     void mouseReleaseEvent(QMouseEvent *e);
-	void enterEvent(QEvent * e);	
+	void enterEvent(QEnterEvent * e);	
 	void leaveEvent(QEvent * e);
 
 private:
@@ -266,4 +266,4 @@ class DkSaturation : public DkFakeMiniat
 		~DkSaturation();
 };
 
-};
\ No newline at end of file
+};
diff -up nomacs-3.17.2295/ImageLounge/plugins/PaintPlugin/src/DkPaintPlugin.cpp.omv~ nomacs-3.17.2295/ImageLounge/plugins/PaintPlugin/src/DkPaintPlugin.cpp
--- nomacs-3.17.2295/ImageLounge/plugins/PaintPlugin/src/DkPaintPlugin.cpp.omv~	2024-06-01 21:22:55.145095545 +0200
+++ nomacs-3.17.2295/ImageLounge/plugins/PaintPlugin/src/DkPaintPlugin.cpp	2024-06-01 21:28:18.792330774 +0200
@@ -30,6 +30,7 @@
 
 #include <QDebug>
 #include <QMouseEvent>
+#include <QActionGroup>
 
 namespace nmp {
 
@@ -480,7 +481,7 @@ QImage DkPaintViewPort::getPaintedImage(
 				//if (worldMatrix)
 				//	painter.setWorldTransform(*worldMatrix);
 
-				painter.setRenderHint(QPainter::HighQualityAntialiasing);
+//				painter.setRenderHint(QPainter::HighQualityAntialiasing);
 				painter.setRenderHint(QPainter::Antialiasing);
 
 				for (int idx = 0; idx < paths.size(); idx++) {
