diff --git a/src/DkImageLoader.cpp b/src/DkImageLoader.cpp
index 5d66e09..3a00010 100644
--- a/src/DkImageLoader.cpp
+++ b/src/DkImageLoader.cpp
@@ -512,6 +512,7 @@ QSharedPointer<DkImageContainerT> DkImageLoader::getSkippedImage(int skipIdx, bo
 
 	}
 
+#ifdef WITH_QUAZIP
 	if (mCurrentImage && (newFileIdx < 0 || newFileIdx >= mImages.size()) && mCurrentImage->isFromZip() && mCurrentImage->getZipData()) {
 
 		// load the zip again and go on from there
@@ -522,6 +523,7 @@ QSharedPointer<DkImageContainerT> DkImageLoader::getSkippedImage(int skipIdx, bo
 
 		return getSkippedImage(newFileIdx);
 	}
+#endif
 
 	// this should never happen!
 	if (mImages.empty()) {
